thirdeye-core/src/test/java/ai/startree/thirdeye/mapper/ApiBeanMapperTest.java

```diff
@@ -133,6 +137,46 @@ public void testToEnumerationItemDTO() {
     assertEquals(gotDto.getAuth().getNamespace(), "my-namespace");
   }
 
+  @Test
+  public void testToAnomalyApi() {
+    final AnomalyDTO dto = new AnomalyDTO();
+    dto.setAuth(new AuthorizationConfigurationDTO().setNamespace("my-namespace"));
+
+    final AnomalyApi gotApi = ApiBeanMapper.toApi(dto);
+    assertNotNull(gotApi.getAuth());
+    assertEquals(gotApi.getAuth().getNamespace(), "my-namespace");
+  }
+
+  @Test
+  public void testToAnomalyDTO() {
+    final AnomalyApi api = new AnomalyApi().setAuth(new AuthorizationConfigurationApi().setNamespace(
+        "my-namespace"));
+
+    final AnomalyDTO gotDto = ApiBeanMapper.toDto(api);
+    assertNotNull(gotDto.getAuth());
+    assertEquals(gotDto.getAuth().getNamespace(), "my-namespace");
```

Comment:
would prefer to use assertj APIs in testing. 

it should be available in imports.
```    assertThat(march21Anomalies.stream()
        .anyMatch(a -> a.getEndTime().getTime() == MARCH_23_2020_00H00)).isTrue();
    assertThat(response.getStatus()).isEqualTo(200);

```
doc: https://assertj.github.io/doc/#overview-what-is-assertj